polymer.externs.js

```diff
@@ -1115,7 +1122,9 @@ var PolymerTouchEvent;
  * @typedef {{
  *   index: number,
  *   removed: !Array,
- *   addedCount: number
+ *   addedCount: number,
+ *   object: !Array,
```

Comment:
I get that, it just seems like it should be called "array". Anyway, your externs are accurate, so wasn't saying anything needs to change here. Just found it weird.